src/App/App.js

```diff
@@ -94,6 +106,10 @@ class App extends Component {
     this.setState( { asideView: view })
   }
 
+  setCardAreaView = (view) => {
+    this.setState( {cardAreaView: view})
+  }
+
```

Comment:
This is a useful method. Previously `cardAreaView` was only being changed by `setDeckPrices()`, but since that property will have to be changed in other places, it's a good idea to have a method for it. 👍